actnow/profiles/validators.py

```diff
@@ -0,0 +1,7 @@
+from django.core.validators import RegexValidator
+
+phone_regex = RegexValidator(
+    regex=r"^\+?1?\d{9,15}$",
+    message="Phone number must be entered in the format: '+999999999'.\
+        Up to 15 digits allowed.",
+)
```

Comment:
Wouldn't using [`PhoneNumberField`](https://github.com/stefanfoulis/django-phonenumber-field) handle all this for us?

Reply:
Felt like overkill to add a whole package just for one field.

Reply:
I hear you but without it, we're pushing the burden of formatting phone numbers to users.